### PR TITLE
Make size of Paragraph section configurable (#5728)

### PR DESCRIPTION
**Pull request by AGMETEOR**
## Description
This PR adds the ability to change the width of the text section in a paragraph section

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [x] Other... Please describe: Enhancement

## Issue

**What is the current behavior?**

#5728 

**What is the new behavior?**
Posible to change the width of text section in paragraph from toolbar

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Review by allyoucanmap — COMMENTED** [review]
@tdipisa @AGMETEOR This implementation adds a size button for each text inside a paragraph as requested in the description of the issue so each text could have a different size, see screentshot:

image

Is this the expected behaviour? Should each text be configurable or should we add a single size button for the whole paragraph section? I'm asking because the title of the issue is referring to paragraph size while the description is referring to texts.

There is also a side effect with this implementation, the columns inside the immersive section have an additional button to shrink the width:

image
 
Before to proceed with additional reviews I would like to have a feedback from @tdipisa.

**Comment by tdipisa** [comment]
@allyoucanmap in my opinion the behavior is correct as long as we have the possibility to configure the default size for the whole paragraph sections.
Anyway, in my opinion paragraphs inside columns should not be resizable: they are resized according to the column size as it is now.

**Review by allyoucanmap — CHANGES_REQUESTED** [review]

File: web/client/components/geostory/contents/Column.jsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ export default ({
             contentToolbar
         }}
         tools={{
-            [ContentTypes.TEXT]: ['remove'],
+            [ContentTypes.TEXT]: ['remove', size(true)],
```
[the remark this answers is not in the document]
Comment (AGMETEOR) [reply web/client/components/geostory/contents/Column.jsx]: @allyoucanmap this is a very good solution. I have tested it. It works. Changed the implementation.

File: web/client/components/geostory/contents/Column.jsx [hunk 2]
```diff
@@ -69,7 +70,8 @@ export default ({
             [MediaTypes.IMAGE]: ['editMedia', size(), 'showCaption', 'remove'],
             [MediaTypes.MAP]: ['editMedia', 'editMap', size(true), 'showCaption', 'remove'],
             [ContentTypes.WEBPAGE]: ['editURL', size(true), 'remove'],
-            [MediaTypes.VIDEO]: ['editMedia', 'muted', 'autoplay', 'loop', 'showCaption', 'remove']
+            [MediaTypes.VIDEO]: ['editMedia', 'muted', 'autoplay', 'loop', 'showCaption', 'remove'],
+            ...overrideTools
```
Comment (allyoucanmap) [inline web/client/components/geostory/contents/Column.jsx:74]: Hi @AGMETEOR, could you please add a test for this new `overrideTools` prop?

**Review by allyoucanmap — CHANGES_REQUESTED** [review]

File: web/client/components/geostory/contents/Column.jsx (hunk 2 above)
Comment (AGMETEOR) [reply web/client/components/geostory/contents/Column.jsx:74]: @allyoucanmap added the test

**Review by allyoucanmap — APPROVED** [review]

**Pull request merged**